bitcoin/src/pow.rs

```diff
@@ -247,19 +256,50 @@ impl Target {
         TARGET_MAX_F64 / self.0.to_f64()
     }
 
+    /// Computes the minimum valid [`Target`] threshold allowed for a block in which a difficulty
+    /// adjustment occurs.
+    #[deprecated(since = "TBD", note = "use min_transition_threshold instead")]
+    pub fn min_difficulty_transition_threshold(&self) -> Self { self.min_transition_threshold() }
+ 
```

Comment:
In 14ab447e37411cde144eb81bad10ba5b425dcf3f:

trailing whitespace.

Reply:
Ha, I've been so busy being annoyed that we don't format in CI anymore that I didn't even look at the git pre-commit hook, I just assumed it was failing because of `rustfmt` and have been overriding it for the last month. Turns out its because we never updated it to run `clippy` with nightly - :(